overlord/snapstate/autorefresh.go

```diff
@@ -368,6 +369,9 @@ func (m *autoRefresh) refreshScheduleWithDefaultsFallback() (ts []*timeutil.Sche
 
 // launchAutoRefresh creates the auto-refresh taskset and a change for it.
 func (m *autoRefresh) launchAutoRefresh() error {
+	perfTimings := timings.New(map[string]string{"ensure": "auto-refresh"})
+	tm := perfTimings.StartSpan("auto-refresh", "prepare auto-refresh change")
```

Comment:
I'm not sure what is the best verb here, "query store and setup auto-refresh change" ?

Reply:
Sounds good.